src/main/resources/hudson/plugins/throttleconcurrents/ThrottleJobProperty/global.jelly

```diff
@@ -4,7 +4,7 @@
   <f:section title="Throttle Concurrent Builds">
     <f:entry title="Multi-Project Throttle Categories" field="categories">
       <f:repeatable field="categories" add="${%Add Category}" minimum="0">
-        <table width="100%">
+        <div style="width:100%">
```

Comment:
This needs either the minimum jenkins version increased or the compatibility code added:
https://www.jenkins.io/doc/developer/views/table-to-div-migration/#maintaining-support-for-tables-and-div-layout